src/components/Navbar.css

```diff
@@ -0,0 +1,3 @@
+@tailwind base;
```

Comment:
We do not need to add it in every css file. Just add it in the index.css or app.css file